### ISSUE TITLE
Inconsistent indentation not detected

### ISSUE THREAD

**Issue by johnvh** [rubocop/rubocop#631]
As of dc7544f825f93fb19b75bcc97b8907afe696c97c, using:

```
$ cat .rubocop.yml
IndentationWidth:
  Enabled: true
```

Then:

```
$ cat test_1.rb && rubocop test_1.rb
if foo
  do_bar
    do_baz
end
Inspecting 1 file
.

1 file inspected, no offences detected

$ cat test_2.rb && rubocop test_2.rb
if foo
    do_bar
  do_baz
end
Inspecting 1 file
C

Offences:

test_2.rb:2:1: C: Use 2 (not 4) spaces for indentation.
    do_bar
^^^^

1 file inspected, 1 offence detected
```

It looks to me like the [IndentationWidth cop](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/indentation_width.rb) should catch this.
